libs/application/template-api-modules/src/lib/modules/templates/marriage-conditions-submission/emailGenerators/assignOtherSpouseEmail.ts

```diff
@@ -0,0 +1,98 @@
+import get from 'lodash/get'
+
+import { ApplicationConfigurations } from '@island.is/application/types'
+import { Message } from '@island.is/email-service'
+
+import { EmailTemplateGeneratorProps } from '../../../../types'
+import { pathToAsset } from './utils'
+
+export let linkOtherSpouseSMS = ''
+
+export type AssignOtherSpuseEmail = (
+  props: EmailTemplateGeneratorProps,
+  senderName?: string,
+  senderEmail?: string,
+) => Message
+
+// TODO handle translations
+export const generateAssignOtherSpouseApplicationEmail: AssignOtherSpuseEmail = (
+  props,
+): Message => {
+  const {
+    application,
+    options: { email, clientLocationOrigin },
+  } = props
+
+  const otherSpouseEmail = get(application.answers, 'spouse.email')
+
+  if (!otherSpouseEmail) {
+    throw new Error('Could not find other Spouse email')
+  }
+
+  const subject = 'Umsókn vegna hjónavígslu'
+  const link = `${clientLocationOrigin}/${ApplicationConfigurations.MarriageConditions.slug}/${application.id}`
+
+  linkOtherSpouseSMS = link
+
+  return {
+    from: {
```

Comment:
Do you know of any examples of this in the code?

Reply:
I can add it in hard coded, would be nice to have this in the `emailOptions` in `BaseTemplateAPIModuleConfig` so that all future applications can be using the same replyTo `umsoknir@island.is`. I'll add it as a task on the application task list in Notion :)